### PR TITLE
File Purge

### PR DESCRIPTION
**Pull request by markhayden**
Removing all the unnecessary template files, js files and general outdate / junk files. Should be left with core async and io files necessary to move forward. Integration with Gulp and other system level changes to be included in separate pr.

### CODE REVIEW

File: out/spec/jstagSpec.js [hunk 1]
```diff
@@ -1,164 +0,0 @@
...
-    runs(function() {
-      // we should not have anything 
-      expect(osent3).toBeNull();
-      expect(io3).toBeNull();
-    });
-  });
-  */
-
-});
-
```
Comment (DingoEatingFuzz) [inline out/spec/jstagSpec.js]: NOT THE TESTS! 😭

**Comment by DingoEatingFuzz** [comment]
It looks like there are still some things that are safe to delete.

  - docpad.coffee
  - index.html
  - serve.go
  - build (at least partially?)

**Comment by DingoEatingFuzz** [comment]
Feel free to tell me not to worry about it because the cleanup PR handles all that.

File: out/spec/jstagSpec.js (hunk 1 above)
Comment (markhayden) [inline out/spec/jstagSpec.js]: they dont run :)

**Comment by markhayden** [comment]
the cleanup pr should handle all of that but docpad, index and serve should all be in this purge one. build gets replaced by gulp in the other one so ill leave that over there.

**Comment by DingoEatingFuzz** [comment]
✨ 👍 ✨

**Comment by markhayden** [comment]
going to close this and pr it against develop so we can get off of gh-pages

**Pull request closed without merging**